### PR TITLE
Deselect button pushed to the right #68

### PR DESCRIPTION
**Pull request by todormoskov**

### CODE REVIEW

File: frontend/src/components/ActionBar.vue [hunk 1]
```diff
@@ -23,27 +23,16 @@
...
-            <div class="section">
-              <svg viewBox="0 3.5 28 18" width="22px" height="24px" display="inline">
-                <path d="M8 9v-4l8 7-8 7v-4h-8v-6h8zm2-7v2h12v16h-12v2h14v-20h-14z"/>
-              </svg>
-              Login
-            </div>
-          </a>
-        </li>
+        <Action
+          v-for="action in rightSideActions"
```
Comment (tailoric) [inline frontend/src/components/ActionBar.vue:30]: image

When pressing the logout button while having files selected the Selected button is still visible.

Not sure how to best fix it but the `v-if="isAuthenticated"` check is not done like in the left side actions.

**Review by tailoric — CHANGES_REQUESTED** [review]
Code looks good but only one small bug that still needs fixing

**Review by Smoothex — APPROVED** [review]
Yup, looks good 👍 bug is fixed and now the Deselect button does not appear after the user is logged out

File: frontend/src/components/ActionBar.vue (hunk 1 above)
Comment (todormoskov) [reply frontend/src/components/ActionBar.vue:30]: Hi Eric, I added a quick fix that checks if the user is authenticated in the setEnabled() method of the Deselect action.

**Review by zlatinametodieva — APPROVED** [review]

**Review by tailoric — APPROVED** [review]

**Comment by tailoric** [comment]
looks good going to merge it now

**Pull request merged**